Bottom: 0c3508a7c794ba62284d659afbb3cc354b651e22
Top:    439fa2c598666a7a3e752c0481e4f07dd355cf22
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 15:42:34 +0000

Refresh of fix-3d-contour-export

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 54ee0f3..0e9ff37 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1157,7 +1157,6 @@ old_object3d_from_board_outline (void)
       DDATA (edges[1 * npoints + i]) = vertices[1 * npoints + next_i_around_ct];
       ODATA (edges[2 * npoints + i]) = vertices[0 * npoints + i];
       DDATA (edges[2 * npoints + i]) = vertices[1 * npoints + i];
-
 #if REVERSED_PCB_CONTOURS
       RDATA (edges[              i]) = faces[i];
       LDATA (edges[              i]) = faces[npoints];
